src/cmake/thirdparty/FindLUA.cmake

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+# Setup Lua
+#------------------------------------------------------------------------------
+# This file defines:
+#  LUA_FOUND - If Lua was found
+#  LUA_INCLUDE_DIRS - The Lua include directories
+#  LUA_LIBRARY - The Lua library
+#------------------------------------------------------------------------------
+
+# first Check for LUA_DIR
+if (NOT EXISTS "${LUA_DIR}")
+    message(FATAL_ERROR "Given LUA_DIR does not exist: ${LUA_DIR}")
+endif()
+
+if (NOT IS_DIRECTORY "${LUA_DIR}")
+    message(FATAL_ERROR "Given LUA_DIR is not a directory: ${LUA_DIR}")
+endif()
+
+# Find includes directory
+find_path( LUA_INCLUDE_DIR lua.hpp
```

Comment:
Who sets `LUA_INCLUDE_DIR`? Should this be `${LUA_DIR}/include` ?

Reply:
It is the output variable of this cmake function.

Reply:
D'oh...that makes sense...